source/deployment/sso-saml-ldapsync.rst

```diff
@@ -52,3 +52,29 @@ When enabled, SAML synchronization with AD/LDAP occurs in phases:
 
  - If any attribute of the user has changed, that attribute is copied from the LDAP server and the user is marked as updated.
  - If the corresponding ``LdapSettings.EmailAttribute`` is not found, the user is assumed to be deleted from the LDAP server, and deactivated from Mattermost by setting the ``Users.DeleteAt`` field to a valid timestamp.
+ 
+Binding SAML to ID Attribute Instead of Email
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ 
+Alternatively, you can choose to use the SAML Id Attribute instead of email to bind the user.  
+
+Configuring this will allow you to reuse an email address for a new user without the old user's information being exposed. For instance, if a user with an email address joe.smith@mattermost.com was once an employee, a new employee named Joe Smith can use the same email. This configuration is also useful when a user has a name change and their email needs to be updated instead of needing to create a new email address.
+
+To bind users to the SAML Id Attribute instead of email: 
```

Comment:
This instructions are not fully true. You can set the IDAttribute and that is, and will work (but won't update old accounts, only will update the accounts while they are logging in in the platform). Besides you need to expose in your SAML server the "id attribute" that you want to use as ID and the idea is that it must be a unique not changing over-time id. (For example: Email or username can change, the ID Integer value for that register in the database won't).

Some users can have saml integration without ldap sync, and they can use the IDAttribute too, without the need of any sync.

Reply:
If we are showing it here because is part of the "SAML LDAP Sync" part, maybe we need a explanation of how IdAttribute works in the SAML configuration section.

Reply:
Added new SAML page; edits made based on feedback provided.